server/src/routes/api/v1/parksRouter.js

```diff
@@ -12,6 +15,20 @@ parksRouter.get("/", async (req, res) => {
   }
 })
 
+parksRouter.post("/", async (req, res) => {
+  const { body } = req
+  const formInput = cleanUserInput(body)
+  try{
```

Comment:
Please clean up whitespace here: spaces between `try {` and `} catch` for example